guideline-07.md

```diff
@@ -1,9 +1,5 @@
-## Do Not Post About Commercial Products
+## Do Not Open Multiple Accounts (Sockpuppets)
 
-For support of commercial themes or plugins, go to the official support channel.
+Do not open multiple accounts, aka [sockpuppets](https://en.wikipedia.org/wiki/Sockpuppet_(Internet)). Sockpuppetry is often used for nefarious reasons, and as such we ask that you simply don’t do it. If a sockpuppet account is identified, it will be [banned](https://make.wordpress.org/support/handbook/forum-welcome/#being-mod-watched-or-banned) immediately.
```

Comment:
"Do not create multiple user accounts" - more clear about what the heck 'multiple accounts' means 😃